src/core/client/admin/components/StoryInfoDrawer/StoryInfoDrawerQuery.tsx

```diff
@@ -33,6 +39,10 @@ const StoryInfoDrawerQuery: FunctionComponent<Props> = ({
           return <QueryError error={error} />;
         }
 
+        if (!props?.viewer) {
+          return <h1>TODO NO VIEWER?!</h1>;
```

Comment:
It looks like this case still needs to be handled. Maybe we show a spinner here too?

Reply:
I changed it to that, but are there cases where viewer is null and it't not because the current user is not logged in? @nick-funk

Reply:
The viewer can be null because you can access one part of the admin (the login page) and have the viewer be unavailable.

If there is no viewer when you're using the StoryInfoDrawerQuery, it is fine to show a spinner or return null. We don't want to be presenting any of the drawer without a valid viewer.

Reply:
aha, good to know